src/slurm_plugin/computemgtd.py

```diff
@@ -114,8 +114,13 @@ def _self_terminate(computemgtd_config):
     instance_manager = InstanceManager(
         computemgtd_config.region, computemgtd_config.cluster_name, computemgtd_config.boto3_config
     )
+    token = check_command_output(  # nosec
```

Comment:
Should we use the `requests` based way to get the metadata here as well? We can put a utility function to get metadata in `common/utils.py` so `nodewatcher` and `computemgtd` can use the same function?

Reply:
From the previous code in `computemgtd` and `nodewatcher`, it looks like  in src/nodewatcher/nodewatcher.py, it is running a python codes to get the metadata.
In `computemgtd` def _self_terminate(computemgtd_config): each commands are execute shell commands and retrieve command output.
I am not sure whether we want to change it be run python code

Reply:
After a second thought, I think it is a good idea to move it and put it in utils, I will test it after change